components/mdns/tests/host_test/sdkconfig.defaults

```diff
@@ -0,0 +1,9 @@
+CONFIG_IDF_TARGET="linux"
+CONFIG_MDNS_NETWORKING_SOCKET=y
+CONFIG_MDNS_SKIP_SUPPRESSING_OWN_QUERIES=y
+CONFIG_TEST_NETIF_NAME="eth0"
+CONFIG_TEST_HOSTNAME="myesp"
+# CONFIG_MDNS_PREDEF_NETIF_STA is not set
```

Comment:
Delete?

Reply:
Well, this is a bit confusing, but cannot delete this. According to the `sdkconfig` spec this means that the `CONFIG_MDNS_PREDEF_NETIF_STA` should be disabled.

Reply:
```suggestion
CONFIG_MDNS_PREDEF_NETIF_STA=n
```